Patterns/English/English-Choices.yaml

```diff
@@ -0,0 +1,9 @@
+﻿---
+LangMarker: Eng
+#Boolean Recognizers
+TokenizerRegex: !simpleRegex
+  def: '[^\w\d]'
+TrueRegex: !simpleRegex
+  def: \b(y(e[sp])?|sure|ok|\uD83D\uDC4D|\uD83D\uDC4C)\b
```

Comment:
add "true" to this

Reply:
The regex would be easier to read and maintain if it was simpler. Specifically avoiding the grouping and flattening the overall structure. So for the true case y|yes|yep reads better than y(e[sp])? same feedback applies for the false case.

Reply:
Changed to improve readability